src/components/ifxRange/ifxRange.scss

```diff
@@ -0,0 +1,101 @@
+@use "~@infineon/design-system-tokens/dist/tokens";
+
+// ifx-range.scss
+:host {
+  display: inline-block;
+}
+
+.dot {
+  display: inline-block;
+  width: 8px;
+  height: 8px;
+  border-radius: tokens.$border-radius-circle;
+}
+
+.left-icon,
+.right-icon,
+.left-text,
+.right-text,
+.percentage-display {
+  font-family: tokens.$font-family-body;
+  font-size: tokens.$font-size-s;
+  color: tokens.$color-gray-500;
+  margin-left: 8px;
+
+  &.disabled { // Add the disabled state for the percentage display
+    color: tokens.$color-gray-300;
```

Comment:
Not sure if it's a Figma mistake or not, but when disabled, the circle should be gray color, and 
`hover` + `active` states should not work.

Reply:
Let Benedict test this